src/platform/forms-system/src/js/patterns/array-builder/helpers.js

```diff
@@ -229,29 +229,30 @@ export function getUpdatedItemFromPath(search = window?.location?.search) {
 
 /**
  * @param {{
- *  arrayPath: string,
  *  nounSingular: string,
  *  nounPlural: string,
- *  maxItems: boolean,
+ *  minItems: number,
+ *  maxItems: number,
  * }} options
  */
-export const maxItemsHint = ({
-  arrayData,
+export const minMaxItemsHint = ({
   nounSingular,
   nounPlural,
+  minItems,
   maxItems,
 }) => {
   let hint = '';
-  const len = arrayData?.length;
 
-  if (maxItems) {
-    if (!len || maxItems === 1 || len >= maxItems) {
-      hint = `You can add up to ${maxItems}.`;
-    } else if (len && maxItems - len === 1) {
-      hint = `You can add 1 more ${nounSingular}.`;
-    } else if (len && maxItems - len > 1) {
-      hint = `You can add ${maxItems - len} more ${nounPlural}.`;
-    }
+  if (minItems && maxItems) {
+    hint += `You need to add a minimum of ${minItems} and maximum of ${maxItems} ${nounPlural}.`;
+  } else if (minItems) {
+    hint += `You need to add a minimum of ${minItems} ${
+      minItems === 1 ? nounSingular : nounPlural
+    }.`;
+  } else if (maxItems) {
+    hint += `You can add up to ${maxItems} ${
+      minItems === 1 ? nounSingular : nounPlural
```

Comment:
As is, this logic will only ever return nounPlural in situations where minItems is undefined, which is the only circumstance in which this path would ever be taken.